mmaction/apis/train.py

```diff
@@ -109,4 +153,11 @@ def train_model(model,
         runner.resume(cfg.resume_from)
     elif cfg.load_from:
         runner.load_checkpoint(cfg.load_from)
-    runner.run(data_loaders, cfg.workflow, cfg.total_epochs)
+    if cfg.omnisource:
+        runner.run(
+            data_loaders,
+            cfg.workflow,
+            cfg.total_epochs,
+            train_ratio=train_ratio)
+    else:
+        runner.run(data_loaders, cfg.workflow, cfg.total_epochs)
```

Comment:
can we use a `runner_kwargs` dict to merge them ?

Reply:
That's OK